autogpt/pyproject.toml

```diff
@@ -111,4 +110,4 @@ ignore = ["../forge/**"]
 
 
 [tool.pytest.ini_options]
-markers = ["slow", "requires_openai_api_key", "requires_huggingface_api_key"]
+markers = ["slow"]
```

Comment:
this should be copied or moved to forge/pyproject.toml, because it is used in test_gcs_file_storage.py

Reply:
I've moved it